src/authorizations/components/TokenRow.tsx

```diff
@@ -43,6 +43,11 @@ class TokenRow extends PureComponent<Props> {
     const {description} = this.props.auth
     const {auth} = this.props
     const labelText = this.isTokenEnabled ? 'Active' : 'Inactive'
+
+    const date = new Date(auth.createdAt)
+    const formatter = createDateTimeFormatter(UPDATED_AT_TIME_FORMAT)
```

Comment:
let's move this out of the render method. since this uses a format that won't change based on user input, we can call this once, before the component is created, to avoid recreating essentially a static formatter on every render loop.

so something like:

```ts
const formatter = createDateTimeFormatter(UPDATED_AT_TIME_FORMAT)
class TokenRow extends PureComponent<Props> {
```